WordPress/Sniffs/Theme/UseCapabilitiesNotRolesSniff.php

```diff
@@ -0,0 +1,231 @@
+		'update_core'            => true,
+		'update_plugins'         => true,
+		'update_themes'          => true,
+		'install_plugins'        => true,
+		'install_themes'         => true,
+		'upload_plugins'         => true,
+		'upload_themes'          => true,
+		'delete_themes'          => true,
+		'delete_plugins'         => true,
+		'edit_plugins'           => true,
+		'edit_themes'            => true,
+		'edit_files'             => true,
+		'edit_users'             => true,
+		'create_users'           => true,
+		'delete_users'           => true,
+		'unfiltered_html'        => true,
+		'delete_others_pages'    => true,
+		'delete_others_posts'    => true,
+		'delete_pages'           => true,
+		'delete_posts'           => true,
+		'delete_private_pages'   => true,
+		'delete_private_posts'   => true,
+		'delete_published_pages' => true,
+		'delete_published_posts' => true,
+		'edit_others_pages'      => true,
+		'edit_others_posts'      => true,
+		'edit_pages'             => true,
+		'edit_posts'             => true,
+		'edit_private_pages'     => true,
+		'edit_private_posts'     => true,
+		'edit_published_pages'   => true,
+		'edit_published_posts'   => true,
+		'manage_categories'      => true,
+		'manage_links'           => true,
+		'moderate_comments'      => true,
+		'publish_pages'          => true,
+		'publish_posts'          => true,
+		'read'                   => true,
+		'read_private_pages'     => true,
+		'read_private_posts'     => true,
+		'upload_files'           => true,
+		'delete_posts'           => true,
+		'delete_published_posts' => true,
+		'edit_posts'             => true,
+		'edit_published_posts'   => true,
+		'publish_posts'          => true,
+		'read'                   => true,
+		'upload_files'           => true,
+		'delete_posts'           => true,
+		'edit_posts'             => true,
+		'read'                   => true,
+	);
+
+	/**
+	 * Process the parameters of a matched function.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @param int    $stackPtr        The position of the current token in the stack.
+	 * @param array  $group_name      The name of the group which was matched.
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array  $parameters      Array with information about the parameters.
+	 *
+	 * @return void
+	 */
+	public function process_parameters( $stackPtr, $group_name, $matched_content, $parameters ) {
+
+		$position = $this->target_functions[ $matched_content ];
+		if ( ! isset( $parameters[ $position ] ) ) {
+			return;
+		}
+
+		$matched_parameter = $this->strip_quotes( $parameters[ $position ]['raw'] );
+		if ( isset( $this->core_capabilities[ $matched_parameter ] ) ) {
+			return;
+		}
+
+		if ( isset( $this->core_roles[ $matched_parameter ] ) ) {
+			$this->phpcsFile->addError(
+				'Use capabilities and not roles in %s().',
+				$stackPtr,
+				'RoleFound',
+				array( $matched_content )
+			);
+		} else {
+			$this->phpcsFile->addWarning(
```

Comment:
I will run the sniff on existing themes and see what turns up and then work from there.